deploy/complete/helm-chart/mushop/charts/storefront/values.yaml

```diff
@@ -15,7 +22,7 @@ resources:
 
 securityContext:
   runAsNonRoot: true
-  runAsUser: 10001
+  runAsUser: 101
```

Comment:
Why 101 , and not 1000 ?

Reply:
New standard for nginx user on unprivileged images.